specs/corla.pvs

```diff
@@ -0,0 +1,923 @@
+%%% model of the Colorado Risk Limiting Audit (CORLA) tool.
```

Comment:
We can't only comment in a new PR because you're limited to just the lines (+/- 3 lines) that are diffs.